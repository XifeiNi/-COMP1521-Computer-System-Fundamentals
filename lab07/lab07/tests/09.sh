./vmsim 3 2 < tests/trace05
