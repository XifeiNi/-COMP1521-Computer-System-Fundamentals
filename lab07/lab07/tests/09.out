
@ t=0, read from pA=0 (vA=0)

PageTable (Stat,Acc,Frame)
[ 0]  L,  0,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=1, read from pA=4 (vA=4)

PageTable (Stat,Acc,Frame)
[ 0]  L,  1,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=2, read from pA=8 (vA=8)

PageTable (Stat,Acc,Frame)
[ 0]  L,  2,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=3, read from pA=12 (vA=12)

PageTable (Stat,Acc,Frame)
[ 0]  L,  3,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=4, read from pA=16 (vA=16)

PageTable (Stat,Acc,Frame)
[ 0]  L,  4,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=5, read from pA=20 (vA=20)

PageTable (Stat,Acc,Frame)
[ 0]  L,  5,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=6, read from pA=24 (vA=24)

PageTable (Stat,Acc,Frame)
[ 0]  L,  6,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=7, read from pA=28 (vA=28)

PageTable (Stat,Acc,Frame)
[ 0]  L,  7,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=8, read from pA=32 (vA=32)

PageTable (Stat,Acc,Frame)
[ 0]  L,  8,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=9, read from pA=0 (vA=0)

PageTable (Stat,Acc,Frame)
[ 0]  L,  9,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

#loads = 1, #saves = 0, #replacements = 0
